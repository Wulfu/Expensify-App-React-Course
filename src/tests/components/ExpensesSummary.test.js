import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render ExpensesSummary correctly with 1 expense' , () => {
  const wrapper = shallow(
    <ExpensesSummary
      expenseCount={1}
      expensesTotal={666}
    />
  );
  expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary correctly with multiple expenses' , () => {
  const wrapper = shallow(
    <ExpensesSummary
      expenseCount={56}
      expensesTotal={6825966}
    />
  );
  expect(wrapper).toMatchSnapshot();
});