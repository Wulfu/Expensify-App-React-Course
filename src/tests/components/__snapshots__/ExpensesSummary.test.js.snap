// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary correctly with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $6.66
  </h1>
</div>
`;

exports[`should render ExpensesSummary correctly with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    56
     
    expenses
     totalling 
    $68,259.66
  </h1>
</div>
`;
